services/m3aggregator/config/aggregator.go

```diff
@@ -104,6 +105,9 @@ type AggregatorConfiguration struct {
 	// EntryCheckBatchPercent determines the percentage of entries checked in a batch.
 	EntryCheckBatchPercent float64 `yaml:"entryCheckBatchPercent" validate:"min=0.0,max=1.0"`
 
+	// Default policies.
+	DefaultPolicies []policy.Policy `yaml:"defaultPolicies" validate:"nonzero"`
```

Comment:
probably should not be validated nonzero here? should provide a defaultPolicies if no configuration is provided?